### PR TITLE
[7.13.x-blue][KIECLOUD-643] - Prepare the blue branch on the rhpam repositories

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: spolti <fspolti@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,17 +14,17 @@ Let’s inspect the rhpam-7-businesscentral cekit module:
 schema_version: 1
 name: "rhpam-7-businesscentral"
 version: "1.0"
-description: "Red Hat Business Central 7.13 installer"
+description: "IBM Business Automation Manager Open Editions Business Central 7.13 installer"
```
Comment (mareknovotny) [inline README.md:17]: we could use only replacement of Red Hat with IBM i think.
Where is the full name use BAMOE but otherwise it is good to replace only necessary refs/names.

File: controller/README.adoc [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Red Hat Process Automation Manager Standalone Controller 7 module
+# Business Automation Manager Open Editions Standalone Controller 7 module
```
Comment (mareknovotny) [inline controller/README.adoc:1]:
```suggestion
# IBM Business Automation Manager Open Editions Standalone Controller 7 module
```

File: dashbuilder/README.adoc [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Red Hat Process Automation Manager Dashbuilder 7 module
+# Business Automation Manager Open Editions Dashbuilder 7 module
```
Comment (mareknovotny) [inline dashbuilder/README.adoc:1]:
```suggestion
# IBM Business Automation Manager Open Editions Dashbuilder 7 module
```

File: dashbuilder/modules/dashbuilder/module.yaml [hunk 4]
```diff
@@ -2,17 +2,17 @@
 schema_version: 1
 name: "rhpam-7-dashbuilder"
 version: "1.0"
-description: "Red Hat Dashbuilder 7.13 installer"
+description: "IBM IBM Business Automation Manager Open Editions Dashbuilder 7.13 installer"
```
Comment (mareknovotny) [inline dashbuilder/modules/dashbuilder/module.yaml:5]:
```suggestion
description: "IBM Dashbuilder 7.13 installer"
```

File: kieserver/README.adoc [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# Red Hat Process Automation Manager KIE Server 7.0 module
+# Business Automation Manager Open Editions KIE Server 7.0 module
```
Comment (mareknovotny) [inline kieserver/README.adoc:1]:
```suggestion
# IBM Business Automation Manager Open Editions KIE Server 7.0 module
```

**Review by mareknovotny — CHANGES_REQUESTED** [review]
I tried to keep consistent replace, so please check my suggestions ;)

File: README.md (hunk 1 above)
Comment (spolti) [reply README.md:17]: sounds good.

**Review by davidesalerno — APPROVED** [review]

**Review by davidesalerno — APPROVED** [review]

File: businesscentral-monitoring/modules/businesscentral-monitoring/module.yaml [hunk 6]
```diff
@@ -1,22 +1,22 @@
 ---
 schema_version: 1
-name: "rhpam-7-businesscentral-monitoring"
+name: "rhpam-8-businesscentral-monitoring"
 version: "1.0"
-description: "Red Hat Business Central Monitoring 7.13 container image"
+description: "IBM Business Central Monitoring 8.0 container image"
```
Comment (AntStephenson) [inline businesscentral-monitoring/modules/businesscentral-monitoring/module.yaml:5]:
```suggestion
description: "IBM Business Automation Manager Open Editions Business Central Monitoring 8.0 container image"
```

File: controller/modules/controller/module.yaml [hunk 7]
```diff
@@ -1,29 +1,29 @@
 ---
 schema_version: 1
-name: "rhpam-7-controller"
+name: "rhpam-8-controller"
 version: "1.0"
-description: "Red Hat Process Automation Manager Standalone Controller 7.13 installer"
+description: "IBM Standalone Controller 8.0 installer"
```
Comment (AntStephenson) [inline controller/modules/controller/module.yaml:5]:
```suggestion
description: "IBM Business Automation Manager Open Editions Standalone Controller 8.0 installer"
```

File: process-migration/modules/process-migration/module.yaml [hunk 8]
```diff
@@ -1,26 +1,26 @@
 ---
 schema_version: 1
-name: "rhpam-7-process-migration"
+name: "rhpam-8-process-migration"
 version: "1.0"
-description: "Red Hat Process Automation Manager Process Migration 7.13 installer"
+description: "IBM Process Migration 8.0 installer"
```
Comment (AntStephenson) [inline process-migration/modules/process-migration/module.yaml:5]:
```suggestion
description: "IBM Business Automation Manager Open Editions Process Migration 8.0 installer"
```

File: kieserver/modules/kieserver/module.yaml [hunk 9]
```diff
@@ -1,22 +1,22 @@
 ---
 schema_version: 1
-name: "rhpam-7-kieserver"
+name: "rhpam-8-kieserver"
```
Comment (Ginxo) [inline kieserver/modules/kieserver/module.yaml:3]:
```suggestion
name: "bamoe-8-kieserver"
```

File: process-migration/modules/process-migration/module.yaml [hunk 10]
```diff
@@ -1,26 +1,26 @@
 ---
 schema_version: 1
-name: "rhpam-7-process-migration"
+name: "rhpam-8-process-migration"
```
Comment (Ginxo) [inline process-migration/modules/process-migration/module.yaml:3]:
```suggestion
name: "bamoe-8-process-migration"
```

File: scripts/update-version.py [hunk 11]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/python3
-# This script will to help to manage rhpam components modules version, it will update all needed files
+# This script will to help to manage ibm bamoe components modules version, it will update all needed files
```
Comment (Ginxo) [inline scripts/update-version.py:2]:
```suggestion
# This script will to help to manage bamoe components modules version, it will update all needed files
```

File: scripts/update-version.py [hunk 12]
```diff
@@ -18,7 +18,7 @@
 
 from ruamel.yaml import YAML
 
-# All rhpam modules that will have the given version updated.
+# All ibm bamoe modules that will have the given version updated.
```
Comment (Ginxo) [inline scripts/update-version.py:21]:
```suggestion
# All bamoe modules that will have the given version updated.
```

File: scripts/update-version.py [hunk 13]
```diff
@@ -45,7 +45,7 @@ def get_shortened_version(version):
 
 def update_version(version, confirm):
     """
-    Update rhpam components module.yaml to the given version.
+    Update ibm bamoe components module.yaml to the given version.
```
Comment (Ginxo) [inline scripts/update-version.py:48]:
```suggestion
    Update bamoe components module.yaml to the given version.
```

File: scripts/update-version.py [hunk 14]
```diff
@@ -85,7 +85,7 @@ def update_version(version, confirm):
 
 def update_main_readme(version, confirm):
     """
-    Update rhpam version on the main README file to the given version.
+    Update ibm bamoe version on the main README file to the given version.
```
Comment (Ginxo) [inline scripts/update-version.py:88]:
```suggestion
    Update bamoe version on the main README file to the given version.
```

File: scripts/update-version.py [hunk 15]
```diff
@@ -115,10 +115,10 @@ def update_main_readme(version, confirm):
 
 
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser(description='RHPAM Version Manager')
+    parser = argparse.ArgumentParser(description='IBM BAMOE Version Manager')
```
Comment (Ginxo) [inline scripts/update-version.py:118]:
```suggestion
    parser = argparse.ArgumentParser(description='BAMOE Version Manager')
```

File: scripts/update-version.py [hunk 16]
```diff
@@ -115,10 +115,10 @@ def update_main_readme(version, confirm):
...
     parser.add_argument('-v', dest='t_version', help='update everything to the next version')
     parser.add_argument('--confirm', default=False, action='store_true', help='if not set, script will not update the '
-                                                                              'rhpam modules. (Dry run)')
+                                                                              'ibm bamoe modules. (Dry run)')
```
Comment (Ginxo) [inline scripts/update-version.py:121]:
```suggestion
                                                                              'bamoe modules. (Dry run)')
```

File: smartrouter/modules/smartrouter/module.yaml [hunk 17]
```diff
@@ -1,26 +1,26 @@
 ---
 schema_version: 1
-name: "rhpam-7-smartrouter"
+name: "rhpam-8-smartrouter"
```
Comment (Ginxo) [inline smartrouter/modules/smartrouter/module.yaml:3]:
```suggestion
name: "bamoe-8-smartrouter"
```

**Review by Ginxo — CHANGES_REQUESTED** [review]
please check my proposals:
The the rest of `ibm-bamoe` values from yamls

wdyt?

File: smartrouter/modules/smartrouter/module.yaml [hunk 18]
```diff
@@ -1,26 +1,26 @@
...
 version: "1.0"
-description: "Red Hat Process Automation Manager Smart Router 7.13 installer"
+description: "IBM Smart Router 8.0 installer"
```
[the remark this answers is not in the document]
Comment (spolti) [reply smartrouter/modules/smartrouter/module.yaml:5]: @mareknovotny asked to use just IBM.

File: README.md [hunk 19]
```diff
@@ -7,28 +7,28 @@ the EAP with the application layer (business central, kieserver, etc.) on it wit
...
 schema_version: 1
-name: "rhpam-7-businesscentral"
+name: "rhpam-8-businesscentral"
 version: "1.0"
-description: "Red Hat Business Central 7.13 installer"
+description: "IBM Business Central 8.0 installer"
 labels:
 - name: "org.jboss.product"
-  value: "rhpam-businesscentral"
+  value: "ibm-bamoe-businesscentral"
```
[the remark this answers is not in the document]
Comment (spolti) [reply README.md:20]: idk :)

File: businesscentral-monitoring/modules/businesscentral-monitoring/module.yaml [hunk 20]
```diff
@@ -25,7 +25,7 @@ ports:
 - value: 8001
 artifacts:
 - name: "rhpam_business_central_monitoring_distribution.zip"
```
Comment (rgdoliveira) [inline businesscentral-monitoring/modules/businesscentral-monitoring/module.yaml:27]: @spolti I suppose the name/md5 will be changed in a different PR?
if so, I suppose `rhpam_business_central_monitoring_distribution.zip` should be renamed to `bamoe_business_central_monitoring_distribution.zip` (also the other zips like add-ons.zip, business-central-eap7-deployable.zip).

In case you need, bamoe CR1 artifacts are already available in candidates folder.

Comment (spolti) [reply businesscentral-monitoring/modules/businesscentral-monitoring/module.yaml:27]: I'll update it after we finish this first renaming step.
